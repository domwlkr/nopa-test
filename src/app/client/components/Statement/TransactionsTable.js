import React from 'react';

const TransactionsTable = (props) => {
  return (
    <table>

    </table>
  );
};

export default TransactionsTable;