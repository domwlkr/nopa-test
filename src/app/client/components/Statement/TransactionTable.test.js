import React from 'react';
import TransactionsTable from './TransactionsTable';
import renderer from 'react-test-renderer';

const singleTransaction = [{'id': 1, 'dateStr': 'Now', 'beneficary': 'PAYPAL ZARA', 'value': '- Â£35.98'}];

test('Renders correctly', () => {
  const transactionsTable = renderer.create(
    <TransactionsTable transactions={singleTransaction} />
  ).toJSON();

  expect(transactionsTable).toMatchSnapshot();
});