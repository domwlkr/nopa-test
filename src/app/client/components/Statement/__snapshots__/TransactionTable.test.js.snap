// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders correctly 1`] = `
<div
  className="table-wrap"
>
  <p>
    Your transactions for the last 30 days
  </p>
  <div
    className="transactions-table"
  >
    <div
      className="transaction-date-wrap"
    >
      <div
        className="transaction-date"
      >
        Today
      </div>
      <div
        className="transaction"
      >
        <span>
          PAYPAL ZARA
        </span>
        <span>
          - Â£35.98
        </span>
      </div>
    </div>
  </div>
</div>
`;
